vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/HypervisorHostHelper.java

```diff
@@ -1215,6 +1218,35 @@ public static DVSSecurityPolicy createDVSSecurityPolicy(Map<NetworkOffering.Deta
         return secPolicy;
     }
 
+    public static DVSMacManagementPolicy createDVSMacManagementPolicy(Map<NetworkOffering.Detail, String> nicDetails) {
+        DVSMacManagementPolicy macManagementPolicy = new DVSMacManagementPolicy();
+        macManagementPolicy.setAllowPromiscuous(false);
```

Comment:
Minor improvement to reduce lines of code:
````
macManagementPolicy.setAllowPromiscuous(
   nicDetails.containsKey(NetworkOffering.Detail.PromiscuousMode) ? 
   Boolean.valueOf(nicDetails.get(NetworkOffering.Detail.PromiscuousMode)) : false);
````
Same for the other properties. Then you won't need the if blocks below.